examples/gke-security-best-practices/builder/platform-uploads/setup-scripts/01-acme-customizations.sh

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash -xe
+
+# skip interactive post-install configuration steps
+export DEBIAN_FRONTEND=noninteractive
+
+sudo apt-get update -y
+sudo apt-get install -y curl software-properties-common
+sudo apt-add-repository -y ppa:ansible/ansible
+
+# Install kubectl
+sudo apt-get update && apt-get install -y apt-transport-https curl
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+cat <<EOF >/etc/apt/sources.list.d/kubernetes.list
+deb http://apt.kubernetes.io/ kubernetes-xenial main
+EOF
+sudo apt-get update
+sudo apt-get install -y kubectl
+
+# Update base
+sudo apt-get update -y
+sudo apt-get -y \
+  -o Dpkg::Options::="--force-confdef" \
+  -o Dpkg::Options::="--force-confold" upgrade
+
+# install base packages
+sudo apt-get install -y --force-yes \
```

Comment:
Why is ansible needed? jq?

Since this is meant to be a "hardened" image, everything installed should be on its own line with an explanation as to why it is needed.

Reply:
1. ansible runs the hardening script. I could start with one of the pre-hardened marketplace images
2. re > everything installed should be on its own line with an explanation as to why it is needed.
Agreed, I will address this.